azuki-runner/src/main/kotlin/com/anaplan/engineering/azuki/runner/ScenarioScriptRunner.kt

```diff
@@ -143,7 +143,8 @@ fun main(args: Array<String>) {
     val imports = File(importsFile).readText()
     val oracleImpls = args.drop(3)
     ScenarioScriptRunner.Log.debug("Starting script runner: testImpl=$testImpl oracleImpls=$oracleImpls")
-    val runner = construct(ScenarioScriptRunner::class, testImpl, oracleImpls, imports)
+    val defaultResultProcessor = construct(ScenarioScriptRunner.ResultProcessor.Default::class)
```

Comment:
Why do we need to construct this explicitly, given
https://github.com/anaplan-engineering/azuki/blob/bcf34e472ecdd99e97631c2f6290f4dfcc54df73/azuki-runner/src/main/kotlin/com/anaplan/engineering/azuki/runner/ScenarioScriptRunner.kt#L30

Reply:
unfortunately, when you call the constructor via reflection default parameters are not respected.. I forget that we hit this issue frequently with the randomized scenario generator